src/test/java/org/mockitousage/constructor/CreatingMocksWithConstructorTest.java

```diff
@@ -83,7 +92,7 @@ public void mocking_inner_classes_with_wrong_outer_instance() {
             fail();
         } catch (MockitoException e) {
             assertThat(e).hasMessage("Unable to create mock instance of type 'InnerClass'");
-            assertThat(e.getCause()).hasMessageContaining("Unable to find a matching 1-arg constructor for the outer instance.");
+            assertThat(e.getCause()).hasMessageContaining("Please ensure that it has a constructor that matches these arguments: [foo]");
```

Comment:
Will change it to print the arg types instead. Adding the available constructors should be pretty easy if you think it's appropriate.